packages/block-library/src/social-links/editor.scss

```diff
@@ -11,7 +11,19 @@
 // When that happens, leverage that instead of the following overrides.
 .editor-styles-wrapper .wp-block-social-link {
 	margin: 0;
-	margin-right: 8px;
+	margin-right: 0.5em;
+
+	> button {
+		padding: 0.375em;
+		line-height: 0;
+		font-size: 1em;
+		display: block;
+
+		svg {
+			width: 1.5em;
```

Comment:
Sorry I'm late to this. I'm not sure we should be changing the social links values away from pixels as all, it doesn't seem like something that should scale with inheritance. To an extent the same with buttons in my opinion, but those I'm more fine with.

But the thing about social links is that the icons displayed, while vector graphics, are scaled to work at specific pixel sizes of 24x24px. If those happen to land on anything other than that, they may be subject to blur.

@kjellr what do you reckon is the best practice with themes here?

Reply:
Social-links are a unique case indeed...
IMO leaving these as px values would be fine.